playground/src/pages/CodingEnvironment.tsx

```diff
@@ -214,9 +212,33 @@ Cycles = [${trace_length.toString()}]`);
       toast.error("Not yet available");
     });
   };
+
+  /**
+   * This verifies the proof that is stored in the session. 
+   * It runs the Rust program that is imported above.
+   * It returns true if the proof is valid, and false otherwise.
+   * As inputs we need program_info, stack_input, stack_output, and proof.
+   */
   const verifyProgram = async () => {
     init().then(() => {
-      toast.error("Not yet available");
+      try {
+
+        if (checkInputs(inputs)) {
+          const result = verify_program(code, inputs, output, proof);
+          setOutput(`
+Verification success with security level of ${result}
+
+`);
+        }
+
+        toast.success("Verification successful");
+
+      } catch (error) {
+      setOutput("Error: Check the developer console for details.");
```

Comment:
This can be updated in a separate PR, but I think we can stop forcing people to use the console and start showing the actual error now that we have toast messages and better error info being passed from `miden-wasm`. Same thing in `runProgram` and `proveProgram`

Reply:
I think I now always set the Output to the corresponding error message.